includes/class-sensei-course.php

```diff
@@ -2779,6 +2780,50 @@ public static function get_courses_page_url() {
 		return apply_filters( 'sensei_course_archive_page_url', $course_page_url );
 	}
 
+	/**
+	 * Get the link to the course completed page.
+	 *
+	 * @since 3.13.0
+	 * @param int $course_id Course ID to append to URL.
+	 *
+	 * @return string The URL or empty string if page is not set or does not exist.
+	 */
+	public static function get_course_completed_page_url( $course_id = 0 ) {
+		$page_id = isset( Sensei()->settings->settings['course_completed_page'] ) ? intval( Sensei()->settings->settings['course_completed_page'] ) : 0;
+		$url     = $page_id ? get_permalink( $page_id ) : '';
+
+		if ( $url && 0 < $course_id ) {
+			$url = add_query_arg( 'course_id', $course_id, $url );
+		}
+
+		/**
+		 * Filter the course completed page URL.
+		 *
+		 * @since 3.13.0
+		 *
+		 * @param string $url Course completed page URL.
```

Comment:
I think we need to add the return here, right?

Reply:
Yup, plus I didn't have the syntax right, plus I added `course_id` as a parameter 😄 . https://github.com/Automattic/sensei/pull/4270/commits/32023b6285064a22600a919b3a78e02fe2068b4f